blueprints/incredible-squaring/blueprint.json

```diff
@@ -57,9 +31,18 @@
   "request_hook": "None",
   "request_params": [],
   "gadget": {
-    "Wasm": {
-      "runtime": "Wasmtime",
-      "sources": []
+    "Native": {
```

Comment:
We should consider adding `blueprint.json` to `.gitignore` since this value is 100% auto-generated.